webServer.js

```diff
@@ -348,7 +348,7 @@ function start() {
 
     var licenses = librarylicensesList(library);
     var assets = libraryAssetsList(library, version);
-    var stargazers_count = null, forks = null, subscribers_count = null;
+    var stargazers_count = null; var forks = null; var subscribers_count = null;
```

Comment:
I don't think should be a change that's occurring, is this something that eslint enforces?
I believe that using commas instead of a new var declaration for each one is cleaner.

cc @PeterDaveHello

Reply:
It was a hangover from when I was trialing eslint's standard guidelines. I've since reset the file and run a `--fix` and it doesn't want to change this line now.